drivers/dma-buf/reservation.c

```diff
@@ -195,8 +195,7 @@ reservation_object_add_shared_replace(struct reservation_object *obj,
 	if (old)
 		kfree_rcu(old, rcu);
 
-	if (old_fence)
```

Comment:
In the commit message please mention the commit ID of the staging branch that you got this from, and the exact commands you used to take the changes from that branch and apply them here.